src/librustdoc/html/static/main.js

```diff
@@ -423,14 +545,14 @@
                             if (lev.pos !== -1) {
                                 elems.splice(lev.pos, 1);
                                 lev_distance = Math.min(lev.lev, lev_distance);
+                                total += lev.lev;
+                                done += 1;
                             } else {
                                 return MAX_LEV_DISTANCE + 1;
                             }
                         }
-                        return lev_distance;
+                        return lev_distance;//Math.ceil(total / done);
```

Comment:
I can see how returning the average would be better, but i'm not sure how effective it will be. A user would need to search for something with multiple generics (which i'm not sure works right now? a search for `Result<File, Error>` returns a lot of bad lev results before any of the functions that return that type show up) before the difference will even show up. I think we can leave this as-is for now and investigate it in the future.

Reply:
I'll leave the comment just in case then if you don't mind.